libraries/plugins/chain/chain_plugin.cpp

```diff
@@ -362,7 +362,17 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
            // private net might be also started with custom genesis file
            else {
               if (options.count("genesis-json") == 0) {
-                 BOOST_THROW_EXCEPTION( std::runtime_error("To start private net, either \"initminer-account-pubkey\" or \"genesis-json\" parameter must be provided.") );
+                 genesis_state_type genesis;
+                 genesis.genesis_time = time_point_sec::from_iso_string("2018-01-01T08:00:00");
+                 genesis.initial_balace = 0;
+                 genesis.initial_public_key = public_key_type(options.at( "initminer-account-pubkey" ).as< std::string >());
+                 genesis.is_private_net = true;
```

Comment:
this is going to fail. -> it is else statement of options.count("initminer-account-pubkey")

Reply:
I've been too tired yesterday evening... Corrected